be/test/storage/lake/primary_key_publish_test.cpp

```diff
@@ -1296,6 +1296,74 @@ TEST_P(LakePrimaryKeyPublishTest, test_index_rebuild_with_dels2) {
     EXPECT_EQ(kChunkSize, read_rows(tablet_id, version));
 }
 
+TEST_P(LakePrimaryKeyPublishTest, test_index_rebuild_with_dels3) {
+    if (!GetParam().enable_persistent_index ||
+        GetParam().persistent_index_type != PersistentIndexTypePB::CLOUD_NATIVE) {
+        return;
```

Comment:
leverage `GTEST_SKIP()` to explicit that the case is skipped under the condition.

Reply:
Will do next